.github/workflows/ci.yaml

```diff
@@ -48,3 +48,27 @@ jobs:
       - name: Run tests
         run: yarn test
 
+  lint:
+    runs-on: ubuntu-latest
+    env:
+      FOUNDRY_PROFILE: ci
+      ETH_RPC_URL: ${{ secrets.ETH_RPC_URL }}
+      INFURA_ID: ${{ secrets.INFURA_ID }}
+      ETHERSCAN_API_KEY: ${{ secrets.ETHERSCAN_API_KEY }}
+      POLYGONSCAN_API_KEY: ${{ secrets.POLYGONSCAN_API_KEY }}
```

Comment:
Do we need these env vars for linting in CI? guessing we can remove them

Reply:
I don't think so